### PR TITLE
The XR Setup page now always sets up the XR settings

### PR DESCRIPTION
**Pull request by Gusinuhe**
### Description

**Fixes**: The XR Setup couldn't finish the process of setting up the desired target due to some timing limitations, in those scenarios, it used to show a pop up with instructions on how to finish the process manually. This PR makes sure the XR Setup page always finishes upsetting the desired XR SDK.

### Type of change

- [x] Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?

Create a new project, import the Creator, and select either Oculus or Windows MR in the XR Setup page. The process must always complete setting the desired target hardware.

### CODE REVIEW

**Comment by tomwim** [comment]
I have the slight feeling that lint does not like "wip"

**Comment by Gusinuhe** [comment]
> I have the slight feeling that lint does not like "wip"

Yeah, I also really like lint 😑

**Comment by Luphoxa** [comment]
I tested this about 6 times and it did not come up. 👍 
As stated in slack, it would be nice if the "Wait until the setup is done" window wouldn't have an "understood" button that needs to be pressed in order to start the process, but instead if the window would stay there until the setup is really done :) 
If it's worth the effort to add this I will leave that decision to you :)

File: Editor/XRUtils/XRLoaderHelper.cs [hunk 1]
```diff
@@ -19,6 +18,7 @@ namespace Innoactive.CreatorEditor.XRUtils
     /// </summary>
     internal class XRLoaderHelper
     {
+        internal const string AttemptedToInitialized = "IsXRLoaderInitialized";
```
Comment (tomwim) [inline Editor/XRUtils/XRLoaderHelper.cs:21]:
```suggestion
        internal const string AttemptedToInitialize = "IsXRLoaderInitialized";
```
I would say a better name is "IsXRLoaderInitialized", though

**Review by tomwim — CHANGES_REQUESTED** [review]

**Comment by tomwim** [comment]
Also tested it and worked for me

**Review by Maxsiom — APPROVED** [review]

**Review by tomwim — APPROVED** [review]

**Pull request merged**